grapher/detailsOnDemand/detailsOnDemand.tsx

```diff
@@ -63,25 +91,35 @@ export class DoDWrapper extends React.Component<{
     render() {
         if (!this.content) return this.props.children
         return (
-            <Tippy
-                content={
-                    <div className="dod-tooltip">
-                        <h3>{this.title}</h3>
-                        <MarkdownTextWrap text={this.content} fontSize={14} />
-                    </div>
-                }
-                interactive
-                hideOnClick={false}
-                arrow={false}
-            >
-                <span
-                    aria-label={`A definition of the term ${this.title}`}
-                    tabIndex={0}
-                    className="dod-term"
+            <span>
+                <Tippy
+                    // Tippy gets it wrong when viewport height is very low
+                    // because it doesn't realize that the .GrapherComponent
+                    // container is overflow: hidden
+                    placement={this.state.innerHeight < 300 ? "right" : "auto"}
```

Comment:
Isn't it enough to just make it `placement="auto"` always, and not worry about resize? I think the problem was that by default it's `placement="top"`.

Tippy does in fact know that `.GrapherComponent` is `overflow: hidden`, it has different behaviour when it's `visible`.

Reply:
This one was strange. I tried just auto initially and it was still glitching out at low heights, but now it's working 🤷